corehq/apps/toggle_ui/views.py

```diff
@@ -94,10 +111,14 @@ def post(self, request, *args, **kwargs):
             item_list = json.loads(item_list)
             item_list = [u.strip() for u in item_list if u and u.strip()]
 
-        affected_users = set(toggle.enabled_users) | set(item_list)
+        old_entries = set(toggle.enabled_users)
+        new_entries = set(item_list)
         toggle.enabled_users = item_list
         toggle.save()
-        for item in affected_users:
+
+        changed_entries = old_entries ^ new_entries  # ^ is XOR in python
+        self.call_save_fn(changed_entries, new_entries)
```

Comment:
You mean use two separate `for` loops in `call_save_fn`, one for newly enabled, and one for newly disabled entries?  I was also thinking of changing the language - I find `old_entries` and `new_entries` a bit confusing, maybe `previously_enabled` and `currently_enabled`?